### DISCUSSION TITLE
GoldHEN Cheats Manager v1.0.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#72] [Announcements]
_dedicated to Leon ~ in loving memory (2009 - 2022)_ :heart:

![](https://github.com/bucanero/apollo-vita/raw/main/data/leon.jpg)

### Added

- Sort Games settings
  - By Name, By Title ID.
- Offline installation for patches
  - Download [patch1.zip](https://github.com/GoldHEN/GoldHEN_Patch_Repository/raw/gh-pages/patch1.zip).
  - Copy `patch1.zip` to root of USB drive. (Do not rename it)
  - Open GoldHEN Cheat Manager and click Update.
  - If patches are installed correctly from USB, you should see the following message:

<details>
<summary>Screenshot (Click to Expand)</summary>

![](https://user-images.githubusercontent.com/37698908/204118861-ae3fa9c1-a429-4bf9-a357-55a8e7e3df77.png)

</details>

- Updating Cheats will now show the cheat database version.

<details>
<summary>Screenshot (Click to Expand)</summary>

![](https://user-images.githubusercontent.com/37698908/204107657-1d62d096-51f4-4e80-80e1-1736bb2e3084.png)

</details>

### Changed

- Improved UI pad controls
- Patches will now be downloaded from the GitHub [Repository](https://github.com/GoldHEN/GoldHEN_Patch_Repository) instead of GitHub Pages. (GitHub Pages will be disabled soon)

### Misc

- Includes latest official cheat+patch pack (2022-12-24)

<hr /><em>This discussion was created from the release <a href='https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/tag/v1.0.2'>GoldHEN Cheats Manager v1.0.2</a>.</em>

**Comment by Pomelo75** [comment]
why some games i got don't show when press triangle? i search manually in database and have correct version and update but star don't appear in side of game i have and of course i can't activa any cheat

**Reply by CallMeJosh0** [reply]
i've got the same issue, i don't know how to fix it
